doc/api_base.rst

```diff
@@ -111,13 +111,13 @@
 
     Perform the resolved transaction. Use the optional `display` object(s) to report the progress. `display` can be either an instance of a subclass of :class:`dnf.callback.TransactionProgress` or a sequence of such instances. Raise :exc:`dnf.exceptions.Error` or dnf.exceptions.TransactionCheckError.
 
-  .. method:: download_packages(pkglist, progress=None)
+  .. method:: download_packages(pkglist, progress=None, callback_total=None)
 
-    Download packages in `pkglist` from remote repositories. Packages from local repositories or from the command line are not downloaded. `progress`, if given, should be a :class:`.DownloadProgress` and can be used by the caller to monitor the progress of the download. Raises :exc:`.DownloadError` if some packages failed to download.
+    Download packages in `pkglist` from remote repositories. Packages from local repositories or from the command line are not downloaded. `progress`, if given, should be a :class:`.DownloadProgress` and can be used by the caller to monitor the progress of the download. `callback_total` is function accepting two parameters - total size of the downloaded content in bytes and time when the download process started in seconds since the epoch. Raises :exc:`.DownloadError` if some packages failed to download.
```

Comment:
> `callback_total` is **a** function accepting two parameters**:** total size of the downloaded content in bytes and time when the download process started**,** in seconds since the epoch.

(the bold highlight didn't work on the punctuation...)